stargate-starter/src/main/java/io/stargate/starter/Starter.java

```diff
@@ -426,6 +431,11 @@ public void start() throws BundleException {
       bundle.start();
     }
 
+    if (startError.get()) {
+      System.out.println("Terminating due to previous service startup errors.");
```

Comment:
Should this be logged at ERROR level?

Reply:
We don't use the logger in starter

Reply:
But maybe it could/should use `System.err`?

Reply:
OSGi normally logs ERROR messages with exception details for these errors.

Reply:
I used `out` because `Starter` all the other startup messages to `out`... just wanted to have this message in the same stream as the rest. I can change it to `err` if you think it's preferable.

Reply:
LogBack probably sends `ERROR` messages to STDOUT too